packages/client/components/PokerEstimateHeaderCardJira.tsx

```diff
@@ -83,8 +84,6 @@ const PokerEstimateHeaderCardJira = (props: Props) => {
   const {stage} = props
   const {serviceTaskId, story} = stage
   const [isExpanded, setIsExpanded] = useState(false)
-  const descriptionRef = useRef<HTMLDivElement>(null)
-  const maxHeight = Math.min(descriptionRef.current?.scrollHeight ?? 300, 300)
```

Comment:
If the user clicks expand, I think we can show 100% of the height of the card. Calculating height like this didn't show the full error description